core/src/stored/mount.cc

```diff
@@ -491,7 +491,7 @@ int DeviceControlRecord::CheckVolumeLabel(bool& ask, bool& autochanger)
         Jmsg(jcr, M_WARNING, 0,
              _("Director wanted Volume \"%s\".\n"
                "    Current Volume \"%s\" not acceptable because:\n"
-               "    %s"),
+               "    %s\n"),
```

Comment:
```suggestion
               "    %s"),
```
`CatalogRequest()` already add `\n` to such messages